telethon/client/chats.py

```diff
@@ -1145,6 +1145,23 @@ async def kick_participant(
         else:
             raise ValueError('You must pass either a channel or a chat')
 
+    async def get_permissions(
+            self: 'TelegramClient',
+            channel: 'hints.EntityLike',
+            user: 'hints.EntityLike'
+    ) -> 'custom.ParticipantPermissions':
```

Comment:
This method needs documentation, similar to how others are documented (method description, parameter description, example code).